modelmapper/misc.py

```diff
@@ -19,6 +21,27 @@
 CHUNK_SIZE = 2048  # The chunk needs to be big enough that covers a couple of rows of data.
 
 
+valid_chars_for_string = set(string.ascii_letters.lower())
+valid_chars_for_integer = set(string.digits)
+
+MAX_DATE_INTEGER = 30001230125959  # year 3000, December 30 12:59:59
```

Comment:
What is this date format called? I have never seen it

Reply:
That is Year Month Day Hour Minute Second without any delimiter between them.